### DISCUSSION TITLE
cannot import name 'get_configurable_parameters' from 'anomalib.config' (unknown location) in 401_nncf.ipynb

### DISCUSSION THREAD

**Discussion by jonghyonyi** [open-edge-platform/anomalib#2470] [Q&A] [answered]
Hi, I'm trying to run anomalib notebooks and confronting a problem with 401_nncf.ipynb.
"cannot import name 'get_configurable_parameters' from 'anomalib.config' (unknown location)" 

I found 'config' module below under anomalib folder is missing. 
Can anybody help with this?

from anomalib.config import get_configurable_parameters

---------------------------------------------------------------------------

ImportError: cannot import name 'get_configurable_parameters' from 'anomalib.config' (unknown location)

**Comment by yxteo2** [comment]
Are you using anomalib 1.0.0 and above?

Try this:
engine, model, datamodule = Engine.from_config(CONFIG_PATH) 

check this command on what you can configure
anomalib train --data MVTec --print_config

**Comment by samet-akcay** [comment, marked as answer]
@jonghyonyi, this notebook may not be up-to-date since it is not included in the CI pipeline. I'll have a look later to confirm if it is broken
